src/main/play-doc/general/MigrationGuide.md

```diff
@@ -20,4 +24,96 @@ To see the license you have, please type the `conduct info` command. This comman
 
 ## Bundle endpoints
 
-Bundle endpoints using `services` instead of `acls` are no longer accepted. `services` were deprecated in 2.0. Please ref to the [ACL documentation](AclConfiguration) for reference on how to express ACLs.
+Bundle endpoints using `services` properties are no longer accepted. `services` were deprecated in 2.0 in favor of ACLs (Access Control Lists).
+
+ACLs have been introduced so that you may have more control over how an endpoint is presented by a proxy. With the previous approach that used `services`, we infered service names for service locator lookups from the first path component, and introduced query parameters in order to declare how a path should be re-written if required. Over time, we found the `services` expression restrictive and unintuitive. The ACL approached (introduced in ConductR 2.0) provides a great deal of flexibility and is more explicit in its declarations.
+
+### How to migrate from `services` to `acls`
+
+Suppose you have an endpoint declared in your `bundle.conf` as:
+
+```
+    endpoints = {
+      "hello" = {
+        bind-protocol = "http"
+        bind-port     = 0
+        services      = ["http://:9000/hello", "http://:9000/api/hello?preservePath"]
+      },
+      "akka-remote" = {
+        bind-protocol = "tcp"
+        bind-port     = 0
+        services      = []
+      }
+    }
+```
+
+... which is the equivalent in sbt:
+
+```
+BundleKeys.endpoints := Map(
+  "hello" -> Endpoint("http", 0, Set(URI("http://:9000/hello"), URI("http://:9000/api/hello?preservePath"))),
+  "akka-remote" -> Endpoint("tcp", 0)
+)
+```
+
+There are three things that are required to be changed:
+
+* service name;
```

Comment:
Within bullet points, don't use `;`, `.`, `and` and start with an uppercase letter. Also, it is not clear for me what the point "service name" exactly means. What about:

```
There are three things that are required to be changed:

* Declaring a service name
* Replace `services` expressions with `acls` expressions
* Rewrite paths for Play and other web servers
```